maintainers/maintainer-list.nix

```diff
@@ -9662,6 +9662,12 @@
     githubId = 39707188;
     name = "Lynn Dong";
   };
+  lyonsyonii = {
```

Comment:
Please, add yourself to maintainer-list.nix as separate commit.

Reply:
Sorry, do you mean a different PR?

I've renamed the commit if this is what you mean.

Reply:
No, just separate commit within this PR. And squash current 5 commits into single one.

Reply:
Done